modules/dnn/perf/perf_einsum.cpp

```diff
@@ -0,0 +1,123 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "perf_precomp.hpp"
+
+namespace opencv_test {
+
+struct EinsumParams {
+    int inputSize;
+    int outputSize;
+    std::string equation;
+    std::vector<MatShape> einsumInpShapes;
+    EinsumParams(std::string equation_, int inputSize_, int outputSize_,  std::vector<MatShape> einsumInpShapes_ = std::vector<MatShape>())
+    {
+        inputSize = inputSize_;
+        outputSize = outputSize_;
+        equation = equation_;
+        einsumInpShapes = einsumInpShapes_;
+    }
+};
+
+static inline void PrintTo(const EinsumParams& params, ::std::ostream* os) {
+     (*os) << "Eqiation=" << params.equation << ", "
+        << "InputSize=" << params.inputSize << ", "
+        << "OutputSize=" << params.outputSize << ", ";
```

Comment:
I think that number of inputs and output is not very informative. Shapes are enough